test/utils/tox/requirements.txt

```diff
@@ -14,3 +14,4 @@ python-systemd
 pycrypto
 botocore
 boto3
+apache-libcloud
```

Comment:
CI does not use this requirements file. Take a look at https://github.com/ansible/ansible/blob/devel/test/runner/requirements/units.txt for requirements for unit tests.